src/common/dom/handle-click.ts

```diff
@@ -0,0 +1,44 @@
+import { HomeAssistant } from "../../types";
+import { LovelaceElementConfig } from "../../panels/lovelace/elements/types";
+import { fireEvent } from "../dom/fire_event.js";
+import toggleEntity from "../../../src/panels/lovelace/common/entity/toggle-entity";
+
+export const handleClick = (
+  node: HTMLElement,
+  hass: HomeAssistant,
+  config: LovelaceElementConfig,
+  hold: boolean
+): void => {
+  let action = config.tap_action || "more-info";
+
+  if (hold && config.hold_action) {
+    action = config.hold_action;
+  }
+
+  if (action === "none") {
+    return;
+  }
+
+  switch (action) {

```

Comment:
Should there be a default case?

Reply:
more-info?

Reply:
Bingo

Reply:
This is actually already handled when action is set:

`let action = config.tap_action || "more-info";`